src/Containers/DropArea/DropArea.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useMemo } from 'react';
+import styled, { useTheme } from 'styled-components';
+import { MainInterface, Main } from '@Utils/BaseStyles';
+import Dropzone, {
+    useDropzone,
+    DropEvent,
+    FileRejection,
+} from 'react-dropzone';
+
+export interface IDropAreaProps
+    extends MainInterface,
+        React.HTMLAttributes<HTMLDivElement> {
+    message?: string;
+    onDragEnter?: React.DragEventHandler<HTMLElement> | undefined;
+    onDragLeave?: React.DragEventHandler<HTMLElement> | undefined;
+    onDropHandler?:
+        | (<T extends File>(
+              acceptedFiles: T[],
+              fileRejections: FileRejection[],
+              event: DropEvent,
+          ) => void)
+        | undefined;
+    isDragEnter?: boolean;
+}
+
+export const DropArea: React.FC<IDropAreaProps> = ({
+    message = 'Drag and drop your files or click here to select',
+    onDragEnter=()=>null,
+    onDragLeave=()=>null,
+    onDropHandler=()=>null,
```

Comment:
Remove word handler it's implied

Reply:
no, becuase it collisions with onDrop attribute of HTML div.

Reply:
Then if it's already there why are you overriding it?

If you still need to override, do onFileDrop